admin_guide/pruning_resources.adoc

```diff
@@ -343,3 +344,152 @@ Such tags tend to have just one image in its history, which effectively prevents
 them from ever being pruned.
 xref:../dev_guide/managing_images.adoc#tag-naming[Learn more about _istag_
 naming.]
+
+[[hard-pruning-registry]]
+== Hard Pruning the Registry
+
+The OpenShift Container Registry can accumulate blobs that are not referenced by
+the {product-title} cluster's etcd. The basic xref:pruning-images[Pruning
+Images] procedure therefore is unable to operate on them. These are called
+_orphaned blobs_.
+
+Orphaned blobs can occur from the following scenarios:
+
+- Manually running the `oc delete image <sha256:image-id>` command only removes
+the image from etcd, but not from the registry's storage.
+- Pushing to the registry initiated by *docker* daemon failures causes some blobs
+to get uploaded but the image manifest (which is uploaded as the very last
+component) does not. All unique image blobs become orphans.
+- An image being refused because of quota suffers similarly to the previous
+example.
+- The standard image pruner may delete an image manifest but be interrupted before
+it deletes the related blobs.
+- There was a bug in the registry pruner which failed to remove blobs that were
+supposed to be removed. Image objects referencing them were removed and the
+blobs became orphans.
+// Find this BZ
+
+_Hard pruning_ the registry is a separate procedure from basic image pruning
+that allows you to remove orphaned blobs. You should hard prune if you are
+running out of storage space in your OpenShift Container Registry and believe
+you have orphaned blobs.
+
+This should be an infrequent operation and is necessary only when you have
+evidence that significant numbers of new orphans have been created. Otherwise,
+you can perform standard image pruning at regular intervals, for example, once a
+day (depending on your number of images being created).
+
+To hard prune orphaned blobs the registry:
+
+. *Log in* to xref:../cli_reference/get_started_cli.adoc#basic-setup-and-login[the CLI] as a user with an
```

Comment:
Chatted with cnegus and @mburke5678 a bit today on this. I was thinking of the style that some of the Atomic docs use where a quick summary of the step is bolded before the full text, e.g.:

https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux_atomic_host/7/html-single/managing_containers/#create_gpg_keys

It's nice to be able to scan the procedure (esp longer ones) and get a feel for the big picture. To @mburke5678's point, it is not consistent in the rest of the topic, though. I had considered that the rest of the sections in this topic are kinda all over the place and wanted to sync them up, but thought it might be outside the scope of this PR. I'd like to do a follow-up after this that makes things more consistent. And to @bfallonf's point below, I removed some of the redundancies between the bolded part and the rest of the step description.